eclass/user-info.eclass

```diff
@@ -37,18 +37,27 @@ egetent() {
 		# lookup by uid/gid
 		local opts
 		if [[ ${key} == [[:digit:]]* ]] ; then
-			[[ ${db} == "user" ]] && opts="-u" || opts="-g"
+			[[ ${db} == "user" ]] && opts=( "-u" ) || opts=( "-g" )
 		fi
 
+		# Handle different ROOT
+		[[ ! -z ${ROOT} ]] && opts+=( -R "${ROOT}" )
+
 		pw show ${db} ${opts} "${key}" -q
 		;;
 	*-openbsd*)
-		grep "${key}:\*:" /etc/${db}
+		grep "${key}:\*:" "${ROOT}/etc/${db}"
```

Comment:
```suggestion
		grep "${key}:\*:" "${EROOT}/etc/${db}"
```

I guess.

Reply:
From talking to people on IRC, I think you need $ROOT here and not $EROOT, since the target system (under $ROOT) will not even check $EPREFIX for users. $EPREFIX != / should output an error for usermod operations IMO and user-info should use $ROOT and not $EROOT

Reply:
@Mord3rca